src/js/utils/dom-data.js

```diff
@@ -3,6 +3,62 @@
  * @module dom-data
  */
 
+import log from './log.js';
+import * as Guid from './guid.js';
+import window from 'global/window';
+
+let FakeWeakMap;
+
+if (!WeakMap) {
```

Comment:
if (!WeakMap) {   //'WeakMap' is undefined  Should be changed to ===> typeof (WeakMap) == "undefined",
And browser also doesn't support 'new Set()'